building/quick_start_qemu.rst

```diff
@@ -0,0 +1,38 @@
+.. _quick_start_qemu:
```

Comment:
Nothing wrong, but I think we already have this here:
https://optee.readthedocs.io/en/latest/faq/faq.html#q-what-is-the-quickest-and-easiest-way-to-try-op-tee

So maybe adding a link to that page from the [build](https://optee.readthedocs.io/en/latest/building/index.html) page would be sufficient?

Reply:
Ah, yes indeed. I think I was expecting a direct access from the main page either under "Getting started" or "Build and run".
Well I'm not sure we should change anything now ;-)